dockstore-webservice/src/main/java/io/dockstore/webservice/core/Author.java

```diff
@@ -17,88 +17,18 @@
 package io.dockstore.webservice.core;
 
 import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
-import java.sql.Timestamp;
-import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
 import javax.persistence.Table;
-import org.hibernate.annotations.CreationTimestamp;
-import org.hibernate.annotations.UpdateTimestamp;
 
 @ApiModel(value = "Author", description = "This describes a non-ORCID author of a version in Dockstore")
 @Entity
 @Table(name = "author")
-public class Author {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @ApiModelProperty(value = "Implementation specific ID for the author in this web service", required = true, position = 0)
-    private long id;
-
-    @Column(columnDefinition = "varchar(255)", nullable = false)
-    @ApiModelProperty(value = "Name of the author", required = true, position = 1)
-    private String name;
-
-    @Column(columnDefinition = "varchar(255)")
-    @ApiModelProperty(value = "Role of the author")
-    private String role;
-
-    @Column(columnDefinition = "varchar(255)")
-    @ApiModelProperty(value = "Affiliation of the author")
-    private String affiliation;
-
-    @Column(columnDefinition = "varchar(255)")
-    @ApiModelProperty(value = "Email of the author")
-    private String email;
-
-    // database timestamps
-    @Column(updatable = false)
-    @CreationTimestamp
-    private Timestamp dbCreateDate;
-
-    @Column()
-    @UpdateTimestamp
-    private Timestamp dbUpdateDate;
+public class Author extends AbstractAuthor {
 
     public Author() {
     }
 
     public Author(String name) {
-        this.name = name;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(final String name) {
-        this.name = name;
-    }
-
-    public String getRole() {
-        return role;
-    }
-
-    public void setRole(final String role) {
-        this.role = role;
-    }
-
-    public String getAffiliation() {
-        return affiliation;
-    }
-
-    public void setAffiliation(final String affiliation) {
-        this.affiliation = affiliation;
-    }
-
-    public String getEmail() {
```

Comment:
Hmmm, this could use a second look. 
Check my assumptions, but if I recall, currently author emails get into the system if 
1) authors add email into workflow metadata like with https://docs.dockstore.org/en/stable/advanced-topics/best-practices/wdl-best-practices.html?highlight=author#authorship-metadata
2) authors add email via github apps https://docs.dockstore.org/en/stable/getting-started/github-apps/github-apps.html#workflow-yml-file 

So far so good. 

However, with OrcidAuthor inheriting this, will this mean that emails get into our system as harvested from the ORCID API? I ask because I don't think emails are normally public in ORCID.